packages/sdk/src/Base.ts

```diff
@@ -162,9 +162,9 @@ class Base {
     }
   }
 
-  sendTransaction (transactionRequest: providers.TransactionRequest, chain: TChain) {
+  sendTransaction (transactionRequest: providers.TransactionRequest, chain: TChain, overrides?: any) {
```

Comment:
it seems redundant to have an overrides parameter here because the overrides should already be includes in the transactionRequest object. I think it can be removed